source/arriving/setup_computer.rst

```diff
@@ -1,35 +1,274 @@
+
+.. code-block:: bash
+
+    export WORKON_HOME=~/Envs/
+    #export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3   # uncomment this line if you have errors when starting your terminal (next step)
+    source /usr/local/bin/virtualenvwrapper.sh
+
+Finally, you can restart the terminal and create an environment:
+
+.. code-block:: bash
+
+    NAME=somename # Ex: $NAME could be my_env_py3
+                    # depending on the use you will give to this environment.
+                    # Try to be explicit
+    # One of:
+    mkvirtualenv $NAME
+    mkvirtualenv --python=python3.10 $NAME   # To use one precise python version
+                                                # You can also use the complete path to the python installation
+    mkvirtualenv $NAME --python=/usr/bin/python3.10  # Equivalent. The python
+                                                        # version must be already
+                                                        # installed on your computer
+
+    # Note. To download a python version to your /usr/bin without installing it:
+    # https://www.python.org/downloads/
+
+When everything is set, restart your terminal. You should now be able to work in a chosen environmnent by using :bash:`workon $NAME`. Now everytime you need to install a new library inside the environment, you should always try to use :bash:`pip install your_library`. The environment will use the right pip based on the current python version. *Don't use sudo pip!* It will use the pip of the system!
 
-1. First, you should work in a virtual environment to avoid breaking your python installation. See the :ref:`ref_venvs`.
+If the library can't be installed via pip, however, you can use :bash:`sudo apt-get install your_library`. In this case the `sudo` will not be a problem.
 
-2. Install scilpy (see the :ref:`ref_scilpy` page). This will install Dipy too.
+You might want to always have a specific environment when opening a terminal. To do so, simply open the .bashrc and copy-paste this :bash:`workon somename` at the end of it (don't forget to replace :bash:`somename` by the actual name of your environment!). Before moving on to the next steps, make sure to have a virtual environment active!
 
-3. You may want to install Tractoflow (see :ref:`ref_tractoflow`) or our other tools (see :ref:`ref_other_pipelines`).
+Git
+"""
 
-4. You may want to install other tools that are often useful for people in our lab (see the :doc:`here <../intro_to/explore_software>` page for MI-brain, Ants, MRtrix, FSL and Freesurfer, etc.) 
+Git is a version control software system often used for versioning of source code by developers. In the lab, we use the website `Github <https://github.com/>`_ to help us store and manage our code using Git in a user-friendly way. You should get used to it from the beginning, as it is a powerful and very helpful tool. If you have never used Git on your computer, the first step would be to set your user.name and user.email:
```

Comment:
Add link to the Git page?

Reply:
Will do when we create the new merged Git page.